src/view/navigation/routes.ts

```diff
@@ -40,15 +43,17 @@ export enum Route {
   LOGIN = 'Login',
   EXPORTS = 'Exports',
   FARMER = 'Farmer',
-  SEARCH_FARMER = 'SearchFarmers',
-  ADD_MILK_ENTRY = 'AddMilkEntry',
-  MILK_ENTRY_DETAILS = 'MilkEntryDetails',
-  EDIT_MILK_ENTRY = 'EditMilkEntry',
-  EDIT_FARMER = 'EditFarmer',
-  ADD_FARMER = 'AddFarmer',
-  ADD_EXPORT_ENTRY = 'AddExportEntry',
-  EXPORT_ENTRY_DETAILS = 'ExportEntryDetails',
-  EDIT_EXPORT_ENTRY = 'EditExportEntry',
+  SEARCH_FARMER = 'Search Farmers',
+  ADD_MILK_ENTRY = 'Add Milk Entry',
+  MILK_ENTRY_DETAILS = 'Milk Entry Details',
+  EDIT_MILK_ENTRY = 'Edit Milk Entry',
+  EDIT_FARMER = 'Edit Farmer',
+  ADD_FARMER = 'Add Farmer',
+  ADD_EXPORT_ENTRY = 'Add Export Entry',
+  EXPORT_ENTRY_DETAILS = 'Export Entry Details',
+  EDIT_EXPORT_ENTRY = 'Edit Export Entry',
+  TRANSACTION_LOGS = 'Transaction Log',
+  WAREHOUSE = 'Warehouse',
 }
```

Comment:
this makes them display properly on the header but will it mess anythign up? i tested it and seems fine